nusight2/src/client/components/chart/line_chart/view.tsx

```diff
@@ -32,11 +32,11 @@ export class LineChart extends Component<
 
     return (
       <>
-        <div className="bg-[#ddd] py-2 px-3">
+        <div className="bg-auto-surface-2 shadow shadow-md py-2 px-3">
           <label className="pr-4 shrink-0">
             Minimum Value
             <input
-              className="bg-white px-2 py-1 rounded-sm ml-2 w-24"
+              className="bg-auto-surface-1 placeholder-auto-secondary shadow-inner focus:outline-none focus:outline-auto-divider px-2 py-1 rounded-sm ml-2 w-24"
```

Comment:
Higher contrast outline when focused. Same suggestion for the other inputs below.

```suggestion
              className="bg-auto-surface-1 placeholder-auto-secondary shadow-inner focus:outline-none focus:outline-blue-500 px-2 py-1 rounded-sm ml-2 w-24"
```